zuulcilint/__main__.py

```diff
@@ -190,12 +191,18 @@ def print_warnings(
         for file_path in warnings["warnings"]["bad_yaml_files"]:
             print(f"{file_path}")
 
-    if n_duplicate:
+    if n_duplicate_jobs:
         zuul_utils.print_bold(f"Duplicate job {severity.value}s:", severity)
-        zuul_utils.print_bold(f"Found {n_duplicate} duplicate jobs", None)
+        zuul_utils.print_bold(f"Found {n_duplicate_jobs} duplicate jobs", None)
         for job in warnings["warnings"]["duplicated_jobs"]:
             print(f"{job}")
 
+    if n_duplicate_semaphore:
+        zuul_utils.print_bold(f"Duplicate semaphore {severity.value}s:", severity)
+        zuul_utils.print_bold(f"Found {n_duplicate_semaphore} duplicate semaphores", None)
+        for semaphore in warnings["warnings"]["duplicate_semaphore"]:
+            print(f"{semaphore}")
```

Comment:
I think it would look better if each semaphore name had a "bullet point" in front.
```suggestion
            print(f"- {semaphore}")
```
But I do see how that would make it harder to copy the names of the duplicated semaphores (and presumedly duplicated jobs as well).

Reply:
Maybe the tool should be able to output the results to a file 🤔

Reply:
I guess this could be a new issue then ;)

Reply:
New issue: #103